### PR TITLE
create addTask function

### PR DESCRIPTION
**Pull request by EricaAnnor**
Added the addTask function

### CODE REVIEW

File: task-manager.dart [hunk 1]
```diff
@@ -46,10 +46,22 @@ class Task {
 
 class TaskManager {
   // Add task method
-  void addTask() {}
+  List<Task> tasks= [];
+
+  void addTask(int id,String name, String description,DateTime dueDate){
+    Task newTask = new Task(id,name,description,dueDate);
```
Comment (BambinoRiya) [inline task-manager.dart]: FYI consider checking if ID is already in task list

**Review by BambinoRiya — CHANGES_REQUESTED** [review]
Consider checking if ID is already in list

**Comment by EricaAnnor** [comment]
> Consider checking if ID is already in list

Have fixed it

**Review by BambinoRiya — COMMENTED** [review]
Love the use of the counter to generate IDs!

**Comment by EricaAnnor** [comment]
> > Consider checking if ID is already in list
> 
> Have fixed it

I added a new feature (counter) that gives every new task a different id hence the ids will never be the same

**Pull request merged**